### PR TITLE
refactor: updated icons

### PR DESCRIPTION
**Pull request by mfranzke**
## Proposed changes

Resolves https://github.com/db-ui/mono/issues/1822

## Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [x] Refactoring (fix on existing components or architectural decisions)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

**Comment by nmerget** [comment]
https://db-ui.github.io/mono/review/1822-icon-update/foundations/icons/overview -> navigation_u_turn isn't working

File: packages/components/src/shared/all-icons.ts [hunk 1]
```diff
@@ -1,299 +1,2 @@
...
...eck","komfort_check_in_circle","multiple_cards","multiple_passenger","my_travel","outward_journey","qr_code","qr_code_scan","return_journey","ticket","ticket_discount","ticket_multiple","ticket_subscription","time_outward_journey","time_return_journey","timetable","travel_insurance","airplane_inverted","bicycle_inverted","bicycle_trailer_inverted","call_a_bike_inverted","car_inverted","carsharing_inverted","electric_locomotive_inverted","electric_scooter_inverted","ev_car_inverted","ferry_inverted","handcart_inverted","ice_inverted","intercity_train_inverted","local_bus_inverted","local_train_inverted","long_distance_bus_inverted","s_bahn_inverted","scooter_inverted","speedboat_inverted","subway_inverted","taxi_inverted","train_and_car_inverted","tram_inverted","walking_fast_inverted","walking_inverted","airplane","bicycle","bicycle_trailer","call_a_bike","car","car_sequence","carsharing","electric_locomotive","electric_scooter","ev_car","ferry","handcart","ice","ice_side_view","intercity_train","local_bus","local_train","long_distance_bus","rail_and_fly","railroad_car","s_bahn","scooter","sev","speedboat","sprinter","subway","taxi","train_and_car","tram","walking","walking_fast"];
```
Comment (nmerget) [inline packages/components/src/shared/all-icons.ts:2]: You don't use git hooks right? Please run prettier over this file

Comment (mfranzke) [reply packages/components/src/shared/all-icons.ts:2]: I've generated it on my localhost, why didn't that happen automatically by husky ?

Comment (mfranzke) [reply packages/components/src/shared/all-icons.ts:2]: Created a new issue for investigation: https://github.com/db-ui/mono/issues/1847

Comment (nmerget) [reply packages/components/src/shared/all-icons.ts]: It should be triggered by `lint-staged` not `xo`

Comment (mfranzke) [reply packages/components/src/shared/all-icons.ts]: than we need to investigate why we've "deactivated" it within https://github.com/db-ui/mono/blob/main/.lintstagedrc.js#L7 in the first place.

**Comment by mfranzke** [comment]
> https://db-ui.github.io/mono/review/1822-icon-update/foundations/icons/overview -> navigation_u_turn isn't working

that's not good …

**Comment by mfranzke** [comment]
> https://db-ui.github.io/mono/review/1822-icon-update/foundations/icons/overview -> navigation_u_turn isn't working

fixed

**Pull request merged**